### PR TITLE
Renamed test directories to module name.

### PR DESCRIPTION
**Pull request by veerendra-simha-garikipati**
Signed-off-by: Veerendra Garikipati <veerendra.garikipati@seagate.com>

# Problem Statement
- Renamed test directory names to module names

# Design
-  NA

# Coding 
-  NA

# Testing 
- [ ] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: 
- [ ] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: 
- [ ] Confirm Testing was performed with installed RPM [Y/N]:  

# Review Checklist 
  Before posting the PR please ensure
- [ ] PR is self reviewed
- [ ] Jira is updated
- [ ] Check if the description is clear and explained. 
- [ ] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: py-utils/test/setup.py [hunk 1]
```diff
@@ -38,20 +38,19 @@
     package_dir={"cortx.utils.test": "."},
     packages=[
         "cortx.utils.test",
-        "cortx.utils.test.kvstore",
```
Comment (sachinpunadikar) [inline py-utils/test/setup.py:41]: Why kvstore is not part of packages ?

**Review by sachinpunadikar — CHANGES_REQUESTED** [review]
Please include kvstore in packages for test rpm

File: py-utils/test/setup.py (hunk 1 above)
Comment (veerendra-simha-garikipati) [reply py-utils/test/setup.py:41]: Hi Sachin,
I have moved the file contents of kvstore directory to test_kv_store directory and renamed test_kv_store to kv_store.

**Review by rahul-tripathi-git — APPROVED** [review]
Looks good to me

**Review by sachinpunadikar — APPROVED** [review]
+1

**Comment by sachinpunadikar** [comment]
test this please

**Pull request merged**